pkgs/applications/window-managers/sway/osd.nix

```diff
@@ -0,0 +1,38 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, pkg-config
+, gtk3
+, gtk-layer-shell
+, libpulseaudio
+}:
+
+rustPlatform.buildRustPackage {
+  pname = "swayosd";
+  version = "unstable-2023-05-09";
+
+  src = fetchFromGitHub {
+    owner = "ErikReider";
+    repo = "SwayOSD";
+    rev = "5c2176ae6a01a18fdc2b0f5d5f593737b5765914";
+    hash = "sha256-rh42J6LWgNPOWYLaIwocU1JtQnA5P1jocN3ywVOfYoc=";
+  };
+
+  cargoHash = "sha256-/va3ES6HqFLh3+YFjyj8/x65wvkW0cU9v3fl6pEQ084=";
+
+  nativeBuildInputs = [ pkg-config ];
+
+  buildInputs = [
+    gtk3
```

Comment:
Doesn't require `wrapGAppsHook` somewhere?

Reply:
No, it's just a very simple gtk layer